website/docs: improve the docs about our Events and logging (#15270)
goauthentik/authentik: website/docs/sys-mgmt/events/logging-events.md

Event logging in authentik is highly configurable; you can define the [retention period](./index.md#event-retention-and-forwarding) for storing and displaying events, configure which exact events should trigger a [notification](./notifications.md), and view low-level details about when and where the event happened.

### Troubleshooting with event logs

For details about troubleshooting using logs, including setting the log level (info, warning, etc.), enabling `trace` mode, viewing past logs, and streaming logs in real-time, refer to [Capturing logs in authentik](../../troubleshooting/logs.mdx).

## Enhanced audit logging (Enterprise)

In the enterprise version, each Event details page in the UI, details about each event are abstracted and displayed in an easy-to-access table, and for any event that involves an object being created or modified, the code `diffs` are displayed as well. This allows you to quickly see the previous and new configuration settings.

> For example, say an authentik administraotr updates a user's email address; the old email address and the new one are shown when you drill down in that event's details.

Reviewer: ```suggestion
For example, if an authentik administrator updates a user's email address, both the old and new email addresses will be displayed in the event's detailed view.
```

Reviewer: I'm not sure I get exactly what you mean, but there's a typo to fix for sure

![](./events-diffs.png)

> Areas of the authentik UI where you can view these audits details are:

Reviewer: ```suggestion
You can view audit details in the following areas of the authentik Admin interface:
```

Ok, this one is interesting. Since both locations are in the adm interface, could we just set the base here instead of repeating it twice? as shown in ^^

Author: Yes I like your version much better but still think we need to say Admin interface in both bullets below.

> - **Admin interface > Dashboards > Overview**: In the **Recent events** section, click the name of an event to view details.

Reviewer: ```suggestion
- **Admin interface > Dashboards > Overview**: In the **Recent events section**, click an event name to view its details.
```

Author: The word "section" doesn't need to be bolded; that's not part of the UI.

> - **Admin interface > Events > Logs**: In the list of events, click the arrow toggle beside the name of the even that you want to view details for.

Reviewer: ```suggestion
- **Admin interface > Events > Logs**: In the event list, click the arrow toggle next to the event you want to view.
```

> ## Viewing events in maps and charts (Enterprise)

Reviewer: ```suggestion
## Viewing events in maps and charts :ak-enterprise
```

This *should* work? I don't have the syntax memorized and I CBA to find it

Author: I can't recall the syntax either... and frankly I prefer the simplicity of how it is now, without the badge...

Reviewer: Technically, we should use the badge. Syntax is good I double checked

---
title: Logging events
---

Logs are a vital tool for system diagnostics, event auditing, user management, reporting, and more. They capture detailed information about each event, including the client's IP address, the user involved, the date and time, and the specific action taken.

Event logging in authentik is highly configurable. You can set the [retention period](./index.md#event-retention-and-forwarding) for storing and displaying events, specify which events should trigger a [notification](./notifications.md), and access low-level details about when and where each event occurred.

### Troubleshooting with event logs

> For guidance on troubleshooting with logs, including setting log levels (info, warning, etc.), enabling `trace` mode, viewing historical logs, and streaming logs in real-time, see [Capturing logs in authentik](../../troubleshooting/logs.mdx).

Author: Aha one I get to push back on @kensternberg-authentik. :) That first comma is being used in structure of "For guidance on troubleshooting with logs, <separate clause>, see our docs.
